### ISSUE TITLE
GraphViz extension not working

### ISSUE THREAD

**Issue by donatelloOo** [trustin/sphinx-binary#1]
Following error occurs when using GraphViz extension with basic example.

Cf. http://www.sphinx-doc.org/en/master/usage/extensions/graphviz.html

For example:
``` dot
digraph foo {
   "bar" -> "baz";
}
```

Outputs:
``` console
# Sphinx version: 1.7.6
# Python version: 3.6.6 (CPython)
# Docutils version: 0.14 
# Jinja2 version: 2.10
# Last messages:
#   writing output... [ 60%] xxx/yyy/features
#   
# Loaded extensions:
#   alabaster (0.7.11) from C:\Users\xxx\AppData\Local\Temp\_MEI136082\alabaster\__init__.pyc
#   sphinxcontrib.plantuml (unknown version) from C:\Users\xxx\AppData\Local\Temp\_MEI136082\sphinxcontrib\plantuml.pyc
#   sphinx.ext.graphviz (1.7.6) from C:\Users\xxx\AppData\Local\Temp\_MEI136082\sphinx\ext\graphviz.pyc
Traceback (most recent call last):
  File "C:\projects\sphinx-binary\build\venv\lib\site-packages\sphinx\cmdline.py", line 304, in main
  File "C:\projects\sphinx-binary\build\venv\lib\site-packages\sphinx\application.py", line 331, in build
  File "C:\projects\sphinx-binary\build\venv\lib\site-packages\sphinx\builders\__init__.py", line 342, in build_update
  File "C:\projects\sphinx-binary\build\venv\lib\site-packages\sphinx\builders\__init__.py", line 403, in build
  File "C:\projects\sphinx-binary\build\venv\lib\site-packages\sphinx\builders\__init__.py", line 440, in write
  File "C:\projects\sphinx-binary\build\venv\lib\site-packages\sphinx\builders\__init__.py", line 449, in _write_serial
  File "C:\projects\sphinx-binary\build\venv\lib\site-packages\sphinx\builders\html.py", line 602, in write_doc
  File "C:\projects\sphinx-binary\build\venv\lib\site-packages\docutils\writers\__init__.py", line 80, in write
  File "C:\projects\sphinx-binary\build\venv\lib\site-packages\sphinx\writers\html.py", line 56, in translate
  File "C:\projects\sphinx-binary\build\venv\lib\site-packages\docutils\nodes.py", line 174, in walkabout
  File "C:\projects\sphinx-binary\build\venv\lib\site-packages\docutils\nodes.py", line 174, in walkabout
  File "C:\projects\sphinx-binary\build\venv\lib\site-packages\docutils\nodes.py", line 174, in walkabout
  File "C:\projects\sphinx-binary\build\venv\lib\site-packages\docutils\nodes.py", line 166, in walkabout
  File "C:\projects\sphinx-binary\build\venv\lib\site-packages\docutils\nodes.py", line 1882, in dispatch_visit
  File "C:\projects\sphinx-binary\build\venv\lib\site-packages\sphinx\ext\graphviz.py", line 316, in html_visit_graphviz
  File "C:\projects\sphinx-binary\build\venv\lib\site-packages\sphinx\ext\graphviz.py", line 278, in render_dot_html
  File "C:\projects\sphinx-binary\build\venv\lib\site-packages\sphinx\ext\graphviz.py", line 240, in render_dot
  File "C:\projects\sphinx-binary\build\venv\lib\site-packages\sphinx\ext\graphviz.py", line 393, in _new_env
NameError: name 'os' is not defined
```

Using sphinx-maven-plugin 2.2.3 with GraphViz 2.38 for Windows.

**Comment by trustin** [comment]
That's interesting. Let me take a look.

**Comment by trustin** [comment]
Thanks for your patience. Will be fixed in 0.5.0.
